conf/machine/votp.conf

```diff
@@ -6,7 +6,6 @@
 #@SOC:
 #@DESCRIPTION: Machine configuration based on x86-64 processor
 
-MACHINE = "votp"
```

Comment:
Not sure it is right. We will test it.

Reply:
I have never seen a MACHINE statement in a yocto MACHINE definition file. What is certain is if you leave it in there you can't add a separate machine definition which includes the original i.e. votp.conf. If you do that the build fails.

Reply:
I confirm we can remove "Machine" safely.